src/components/text-editor/examples/text-editor.tsx

```diff
@@ -4,25 +4,22 @@ import { Component, h, State } from '@stencil/core';
  * A basic text editor example.
  */
 @Component({
-    tag: 'limel-example-basic-text-editor',
+    tag: 'limel-example-text-editor',
```

Comment:
Is it OK if we call the example file `examples/text-editor-basic.tsx` and the tag `limel-example-text-editor-basic`?

I know we don't do that _everywhere_ yet, but I'm trying to work our way there.

(I'm happy to make the change myself, I just want to know if that's a change you would be OK with? 😊)

Reply:
I'm fine either ways. But I think in most of our components we don't call the basic (first) example, "basic" 😄 So I tried to keep it consistent with them. But whatever works. We should just decide one day on all of these conventions and follow them. Also update all so that new things can follow the old conventions.

Reply:
I made an "executive decision" years ago, but haven't had the time to go around and update every single instance. You can do it for me if you want to! 😝

Reply:
did it for this one.